test/issues/issues-test.js

```diff
@@ -624,6 +632,7 @@ describe("issues", function() {
         function Foo() {
             return;
         }
+        // eslint-disable-next-line mocha/no-setup-in-describe
```

Comment:
It detects all the times we try to [work around mocha](https://github.com/sinonjs/sinon/pull/2091/commits/b16db80e9577f16fb5c1e6af92dc198dec6598f3) instead of using it.